Lib/test/test_posix.py

```diff
@@ -1539,6 +1540,12 @@ def test_open_file(self):
                                 [sys.executable, '-c', script],
                                 os.environ, file_actions)
         self.assertEqual(os.waitpid(pid, 0), (pid, 0))
+
+        max_tries = 3
```

Comment:
For tests, I prefer to use a timeout rather than a number of tries. Like:

deadline = time.monotonic()
while 1:
  if time.monotonic() > deadline: ... timeout ...
  ... the code ...

Reply:
Changed in 28a8ced